src/routes/dashboard/components/ChannelHealth.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+  import { Column, Grid, Row, Tile } from 'carbon-components-svelte'
+  import { getChannelHealth } from '../helpers/channels'
+  import type { Questions } from '../types'
+
+  export let filteredQuestions: Map<string, Questions>
+
+  let data = getChannelHealth(filteredQuestions.get('aggregate')!)
+  console.log(data)
+
+  $: data = getChannelHealth(filteredQuestions.get('aggregate')!)
+</script>
+
+<svelte:head>
+  <title>Discord Metrics Dashboard</title>
+</svelte:head>
+
+<Grid class="styled-col" style="justify-content:center">
+  <h1 class="number-text">Channel Health</h1>
+  <div style="max-height:100vh; overflow-y:scroll">
+  <Column>
+    {#each Object.values(data) as channel}
+      <Row style="background:{channel.color}" class="list-item">
+        <Column><h4>{channel.channel}</h4></Column>
+        <Column>
+          <Row><h2>{channel.percent}%</h2></Row>
+          <Row><p>answered</p></Row>
+        </Column>
+        <Column>
+          <Row><h2>{channel.unanswered}</h2></Row>
+          <Row><p>questions unanswered</p></Row>
+        </Column>
+      </Row>
+    {/each}
+  </Column>
+</div>
+</Grid>
+
+<style>
+  :global(.list-item) {
+    position: relative;
+    left: unset;
+    bottom: unset;
+    right: unset;
+    margin: 6px;
+    padding: 4px 6px 4px 6px;
+  }
+
+
+::-webkit-scrollbar-track {
+  display: none;
+}
+
+::-webkit-scrollbar-track:hover {
+  display: none;
+}
```

Comment:
what is this for? how does this page compare visually to FireFox?

Reply:
Without it there was a white rectangle on the side where the scrollbar track was

Reply:
oh that's weird, do you have a screenshot of that by chance?

Reply:
ok now I can't replicate it so taking it out :/